### PR TITLE
docs(ecopass-kit): Updated the EcoPass KIT Software Development View 

### PR DESCRIPTION
**Pull request by matbmoser**
## Why do we create this PR?

We want to update the software development view from the Eco Pass kit for release 24.05.

## What we want to achieve with this PR?

Have the software development view referencing the other KITs from Tractus-X in the list of standards.
Updated latest functionalities added to the KIT reference implementation.

### Added
- Added reference to other Tractus-X KITs in the list of standards and prerequisites in the Software Development View
- Added single api retrival flow and api documentation from the DPP reference implementation
- Added reference to the digital product pass verification feature coming in 24.05
- Added description of digital product pass addons

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs. (NO DEPENDENCIES)
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files
- [x] LINTING issues are resolved.
- [x] CHANGELOG updated

### CODE REVIEW

**Review by LuisRickert — APPROVED** [review]

**Comment by matbmoser** [comment]
Ok I checked for the linting and now it it is working.

File: docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md [hunk 1]
```diff
@@ -315,6 +400,14 @@ The following represents a collection of relevant documentation regarding the pr
                   }
               ]
           }
+      },
+      {
+        "key" : "assetLifecyclePhase",
+        "value": "AsBuilt"
+      },
```
Comment (arnoweiss) [inline docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md:407]: This is not mandated by the industry core for R24-05 anymore. Leave it in only if you want to go beyond the compatibility guarantees of CX.

Comment (LuisRickert) [reply docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md:407]: @matbmoser should we leave this is ?

File: docs-kits/kits/Eco_Pass_KIT/changelog.md [hunk 2]
```diff
@@ -34,19 +48,19 @@ All notable changes to this Kit will be documented in this file.
...
-### Updated
+### 24.03 Updated
 
 - Updated battery pass model to latest version `v4.0.0`
 - Updated Business Architecture Diagram
 
 ## [0.1.0] - 2023-08-11
 
-### Added
+### 23.08 Added
```
Comment (HFocken) [inline docs-kits/kits/Eco_Pass_KIT/changelog.md:58]: Please change to 23.09 (also below) - 23.08 does not exist.

**Review by HFocken — CHANGES_REQUESTED** [review]
Thank you - minor detail change required, then good to go!

**Review by danielmiehle — APPROVED** [review]
LGTM

File: docs-kits/kits/Eco_Pass_KIT/changelog.md (hunk 2 above)
Comment (LuisRickert) [reply docs-kits/kits/Eco_Pass_KIT/changelog.md]: Done

File: docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md (hunk 1 above)
Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md:407]: You can remove it, we need just the digital twin type, that's mandatory

**Review by HFocken — APPROVED** [review]
Thank you for addressing the change requests!

**Comment by arnoweiss** [comment]
Y'all are still failing the dash-licence and markdown-lint actions. Also, when building the website, there's a lot of warnings wrt to figures in the eco-pass-kit [1]. Is this addressed in this PR?

[1]

image

File: docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md [hunk 3]
```diff
@@ -53,9 +53,9 @@ In order to retrieve data in the Catena-X Network a number of services need to b
...
 | Digital Product Pass  | The [**EcoPass KIT**] reference implementation. The application responsible for retrieving the passports and interacting with the services listed above.                                                                | [eclipse-tractusx/digital-product-pass](https://github.com/eclipse-tractusx/digital-product-pass)                                                                                                                                                                                                            | CX - 0096                                                   |
```
Comment (jSchuetz88) [inline docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md:59]: The CX-Identifier (CX-0096) should be updated to "CX-0143" (Use Case Circular Economy - Digital Product Passport Standard 1.0.0) as the successor of CX-0096 in R24.05.

**Review by jSchuetz88 — APPROVED** [review]
Thank you for the update of the KIT, looks good to me. Only one minor remark.

**Comment by danielmiehle** [comment]
Thanks for the updates. Can you please check the linting issues? We would like to merge the PR today.

**Comment by matbmoser** [comment]
> Thanks for the updates. Can you please check the linting issues? We would like to merge the PR today.

On my way, forgot to push it

**Comment by matbmoser** [comment]
Now it should be fine :)

**Comment by matbmoser** [comment]
> Y'all are still failing the dash-licence and markdown-lint actions. Also, when building the website, there's a lot of warnings wrt to figures in the eco-pass-kit [1]. Is this addressed in this PR?
> 
> [1]
> 
> image

This is from adoption view no?

File: docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md (hunk 3 above)
Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md]: done!

**Review by danielmiehle — APPROVED** [review]
LGTM - ready for merge!

**Review by maximilianong — APPROVED** [review]

**Pull request merged**